### PR TITLE
Corrected wrong answers

### PR DESCRIPTION
**Pull request by AbdulazizAlhasan**
Corrected wrong answers

# Your Details

- Your Name: Abdulaziz Alhasan
- Your City: London
- Your Slack Name: Abdulaziz Alhasan

# Homework Details

- Module: Git and Github
- Week:1

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -86,25 +90,25 @@ You send the changes of your local repository to Github
...
 
-Send you code to Github
+Saving changes locally
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual studio
```
Comment (ybayraktutan) [inline homework.md:105]: I think this should be VS Code

**Review by ybayraktutan — COMMENTED** [review]
Hi Abdulaziz, Great job with the homework, I just made one suggestion in case you want to use it.

File: homework.md (hunk 1 above)
Comment (AbdulazizAlhasan) [reply homework.md:105]: image
Thank you for the feedback, the full name of the software is Visual Studio code (VS Code)
:)

**Review by oksygenn — COMMENTED** [review]
Hi Abdulaziz! Great descriptive answers! My only suggestion is to change the title to "London Class 7 - Abdulaziz Alhasan - Git - Week 1".  Thanks

File: homework.md [hunk 2]
```diff
@@ -4,38 +4,38 @@
...
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website to look better
```
Comment (ademalagoz) [inline homework.md:31]: If you give some more details it would be better/like colour, layout etc..

Comment (ademalagoz) [reply homework.md:31]: One more thing. As teachers said you should change you title like below.
London Class 7 - Adem Alagoz - Git&GitHub - Week 1

**Pull request closed without merging**